starter-code/index.html

```diff
@@ -3,40 +3,58 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <div id='wrapper'>
+    <div id='nav'>
+      <img src="./images/spotify-logo.png" class='spotify-logo'>
+      <ul class='nav-items'>
+        <li>Premium</li>
+        <li>Discover</li>
+        <li>Help</li>
+        <li>Download</li>
+      </ul>
+    </div>
+    <div class='landing-background'>
+      <div id='landing-text'>
+        <h1>Music for everyone.</h1>
+        <p>Spotify is now free on mobile, tablet and computer.</p>
+        <p>Listen to the right music, wherever you are.</p>
+      </div>
+    </div>
+    <div id='features'>
+      <p class='main-features-title'><span>What’s on Spotify?</span></p>
+      <div class='main-features'>
+        <div id='feature1'>
```

Comment:
Set the same class for the three div tags and you will be able to reuse the code instead of writing the same styling for all of them